packages/react-component-library/src/components/SelectE/partials/StyledInlineButton.tsx

```diff
@@ -0,0 +1,72 @@
+import styled, { css } from 'styled-components'
+import { selectors } from '@defencedigital/design-tokens'
+
+import { COMPONENT_SIZE, ComponentSizeType } from '../../Forms'
```

Comment:
Hmm I think I need to refactor `SwitchE` to utilise this.

Reply:
Yep there's an issue for that: https://github.com/defencedigital/mod-uk-design-system/issues/2827